content/webapp/services/prismic/fetch/exhibitions.ts

```diff
@@ -146,7 +147,6 @@ export const fetchExhibitionRelatedContent = async (
     ...contributorsFields,
     ...eventSeriesFields,
     ...eventPoliciesFields,
-    ...contributorsFields,
```

Comment:
Is this part of an extra cleanup of unused fields? (was it removed on purpose)

Reply:
Not so much unused, as duplicated – look three lines up, it's used twice.

Reply:
ahhh yes, getting cross eyed!